src/lyrics/lyricsfetchersearch.cpp

```diff
@@ -133,6 +133,8 @@ void LyricsFetcherSearch::AllProvidersFinished() {
   if (!results_.isEmpty()) {
     qLog(Debug) << "Using lyrics from" << results_.last().provider << "for" << request_.artist << request_.title << "with score" << results_.last().score;
     emit LyricsFetched(id_, results_.last().provider, results_.last().lyrics);
+  } else {
+    emit LyricsFetched(id_, nullptr, nullptr);
```

Comment:
```suggestion
    emit LyricsFetched(id_, QString(), QString());
```

Reply:
`lyrics` is not a pointer, but a reference so `nullptr` should not be used, `QString::isEmpty()` covers both empty null strings.